src/stream/src/executor/monitor/streaming_stats.rs

```diff
@@ -751,6 +762,78 @@ impl StreamingMetrics {
         )
         .unwrap();
 
+        let log_store_first_write_epoch = register_int_gauge_vec_with_registry!(
```

Comment:
All the newly added metrics are bound to executor_id, do we drop them when the actor/executor is dropped?

Reply:
I implemented a `LabelGuardedXxx`, which will remove the label when it is dropped.